chainerx_build_helper.py

```diff
@@ -45,7 +45,7 @@ def build_extension(self, ext):
         elif self.debug:
             # Being built with `python setup.py build --debug`
             build_type = 'Debug'
-        elif os.getenv('READTHEDOCS', None) == 'True':
+        elif os.getenv('CHAINER_READTHEDOCS', None) == 'True':
```

Comment:
How about using `'0'` / `'1'` for the value instead of `'True'`? (just for consistency with other environment variables)

Reply:
Or is it better to keep the consistency with the `READTHEDOCS` variable?

Reply:
Fixed and configured `CHAINER_READTHEDOCS=1` on RTD.